funcr/funcr.go

```diff
@@ -245,7 +245,7 @@ func newFormatter(opts Options, outfmt outputFormat) Formatter {
 		prefix:       "",
 		values:       nil,
 		depth:        0,
-		opts:         opts,
+		opts:         &opts,
```

Comment:
good optimization :)

Reply:
Probably should be merged separately, I just happened to do it last when I noticed how much slower the last approach became compared to the previous one :grin: